src/CredentialsLoader.php

```diff
@@ -73,9 +73,14 @@ private static function isOnWindows()
     public static function fromEnv()
     {
         $path = getenv(self::ENV_VAR);
-        if (empty($path)) {
+        if (!$path) {
```

Comment:
why did this change from `empty`? Let's keep it the way it was unless there's a specific reason to change it